src/libpgexporter/configuration.c

```diff
@@ -1090,6 +1113,17 @@ pgexporter_reload_configuration(void)
       goto error;
    }
 
+   if (strlen(reload->metrics_path) > 0)
```

Comment:
Put this below the `admin` reload code, and do it the same way -- there is no `sd_notify` and `exit` here...

Reply:
OK! I have changed it.